src/niscope/system_tests/test_system_niscope.py

```diff
@@ -557,32 +559,3 @@ def test_error_message(self, grpc_options):
             assert e.code == -1074118609
             assert e.description.find('Simulation does not support the selected model and board type.') != -1
 
-    def test_get_self_cal_last_date_time(self, single_instrument_session):
-        with pytest.raises(NotImplementedError) as exc_info:
-            single_instrument_session.get_self_cal_last_date_and_time()
-        assert exc_info.value.args[0] == 'cal_fetch_date is not supported over gRPC'
-        assert str(exc_info.value) == 'cal_fetch_date is not supported over gRPC'
-
-    def test_get_ext_cal_last_date_time(self, single_instrument_session):
-        with pytest.raises(NotImplementedError) as exc_info:
-            single_instrument_session.get_ext_cal_last_date_and_time()
-        assert exc_info.value.args[0] == 'cal_fetch_date is not supported over gRPC'
-        assert str(exc_info.value) == 'cal_fetch_date is not supported over gRPC'
-
-    def test_get_self_cal_last_temperature(self, single_instrument_session):
-        with pytest.raises(NotImplementedError) as exc_info:
-            single_instrument_session.get_self_cal_last_temp()
-        assert exc_info.value.args[0] == 'cal_fetch_temperature is not supported over gRPC'
-        assert str(exc_info.value) == 'cal_fetch_temperature is not supported over gRPC'
-
-    def test_get_ext_cal_last_temperature(self, single_instrument_session):
-        with pytest.raises(NotImplementedError) as exc_info:
-            single_instrument_session.get_ext_cal_last_temp()
-        assert exc_info.value.args[0] == 'cal_fetch_temperature is not supported over gRPC'
-        assert str(exc_info.value) == 'cal_fetch_temperature is not supported over gRPC'
-
-    def test_configure_ref_levels(self, single_instrument_session):
```

Comment:
The ones above are supported now, so I understand why they're no longer checking for 'not supported'. But `configure_ref_levels` is still unsupported, so shouldn't this test remain?

Reply:
Added this test back in, I just missed that it wasn't added

Reply:
Looks good.